tests/system/general/test_35_MaxwellCircuit.py

```diff
@@ -115,3 +115,27 @@ def test_08_import_netlist(self):
         self.aedtapp.insert_design("SchematicImport")
         self.aedtapp.modeler.schematic.limits_mils = 5000
         assert self.aedtapp.create_schematic_from_netlist(self.netlist_file1)
+
+    def test_09_create_voltage_source(self):
+        name_type = ["voltage_source", 123, 1.23]
+        for item in name_type:
+            self.component = self.aedtapp.modeler.schematic.create_v_sin(name=item, value=3.14, angle=90)
+            assert self.component.parameters["Name"] == item
+        assert self.component.parameters["Va"] == 3.14
+
+    def test_10_create_current_source(self):
+        name_type = ["current_source", 456, 4.56]
+        for item in name_type:
+            self.component = self.aedtapp.modeler.schematic.create_i_sin(name=item, value=2.72, angle=90)
+            assert self.component.parameters["Name"] == item
+        assert self.component.parameters["Ia"] == 2.72
+
+    def test_11_create_page(self):
+        assert self.aedtapp.create_page("string_test") == True
+        assert self.aedtapp.create_page(123) == True
+        assert self.aedtapp.create_page(3.14) == True
+        assert not self.aedtapp.create_page(["create_page_test"])
+
+    def test_12_get_num_pages(self):
```

Comment:
Could you please add a test on the return value and not only on the value type ?

Reply:
The IronPython method for 'CreatePage' returns only the default None when run.
I don't think it's possible to return other attributes such as the created page name from a method that doesn't have these attributes.(Or it may well be something I've not figured out yet in python...)

Similar methods such as 'Selectpage' and 'DeletePage' similary return None and not the page name.
I've added the nb_pages property after every page is created in the unit test.
So we are at least checking these pages are created whilst not being able to check the names of them.